doc/source/security/selinux.rst

```diff
@@ -0,0 +1,100 @@
+.. _selinux:
+
+===========================================
+Enabling SELinux 
+===========================================
+
+SELinux helps protects the user's home directory data. SELinux provides the following:
+
+* Prevents unauthorized users from exploiting the system
+
+* Allows authorized users to access files 
+
+* Used as a role-based access control system
+
+For more information, see `Percona Server and SELinux <https://www.percona.com/doc/percona-server/LATEST/security/selinux.html>`_
+
+Red Hat and CentOS distributes profiles to extend the SELinux policy module for mysqld. These profiles allow PXC-specific operations. They are:
+
+* percona-xtradb-cluster - extends the PS profile and allows the execution of SST scripts
+
+* wsrep-sst-xtrabackup-v2 - allows execution of the xtrabackup-v2 SST script
+
+Adjusting Policies
+------------------
+
+To adjust PXC-specific configurations, especially SST/IST ports, use the following procedures as ``root``:
+
+To enable port ``4567`` which allows group communication:
+
+Find the tag associated with the ``4567`` port:
+
+.. sourcecode:: bash
+
+    $ semanage port -l | grep 4567 
+    tram_port_t tcp 4567
+
+Find the rules which grant mysqld access to the port:
+
+.. sourcecode:: bash
+
+    $ sesearch -A -s mysqld_t -t tram_port_t -c tcp_socket
+    Found 5 semantic av rules:
+        allow mysqld_t port_type : tcp_socket { recv_msg send_msg } ;
+        allow mysqld_t tram_port_t : tcp_socket { name_bind name_connect } ;
+        allow mysqld_t port_type : tcp_socket { recv_msg send_msg } ;
+        allow mysqld_t port_type : tcp_socket name_connect ;
+        allow nsswitch_domain port_type : tcp_socket { recv_msg send_msg } ;
+
+
+If you require other ports to be added, use the general mysqld tag. For example, to use port 14567, you add the port to the policy module with the ``mysqld_port_t`` tag.
+
+.. sourcecode:: bash
+
+    $ semanage port -a -t mysqld_port_t -p tcp 14567
+
+You can verify the addition with the following command:
+
+.. sourcecode:: bash
+
+    $ semanage port -l | grep 4568
```

Comment:
semanage port -l | grep 14567

Maybe it is worth mentioning why we didn't tag it with tram_port_t?

Reply:
Rewritten